README.md

```diff
@@ -1,43 +1,36 @@
 # GOV.UK Display Screen
 
-Screen to display live data from GOV.UK. Includes number of people on GOV.UK,
-latest searches, trending and recent content. Not publicly accessible because
-there's sometimes personal data in the latest searches.
+Screen to display live data from GOV.UK. It displays the number of GOV.UK users in last 30 mins, live searches, 
+most popular pages and recent content. It is intended as an internal tool only because the live searches are unfiltered.
 
 ![screenshot](docs/screenshot.png)
 
 http://govuk-display-screen.herokuapp.com/
 
 ## Running locally
 
-To run the server you will need the following `ENV` variables set:
+To run the server you will need to set up some `ENV` variables. 
 
-```
-CLIENT_ID
-CLIENT_SECRET
-REFRESH_TOKEN
-```
-
-You can create the client id and client secret using the [Google developer
-console][1]. You need to generate the refresh token, there is a [short Ruby
-script to create one for you][2].
+We are using the [dotenv gem][1] to load environment variables from a .env file into ENV in development. You will need to create
+a .env file, add it into .gitignore, and then add in the keys for the env vars which can be acquired from the 2nd line Heroku account.
+You can access this account by using the 2nd line credentials held in [govuk-secrets][2].
 
-Or, you can [obtain these values from Heroku][3]. For this approach you will need
-to log into Heroku using the 2nd line credentials held in [govuk-secrets][4].
-
-Once you have them you can start the server running:
+Once you have them, you should be able to start the server by running:
 
 ```
-CLIENT_ID=... CLIENT_SECRET=... REFRESH_TOKEN=... ruby ./server.rb
+ruby ./server.rb
 ```
 
-You can then browse to the server in your browser.
+You can then navigate to the server in your browser.
```

Comment:
Could simplify to: You can now navigate to the dashboard in your browser: http://localhost:4567/.